.gitignore

```diff
@@ -10,3 +10,4 @@ functions/lib/
 serviceAccount.json
 serviceAccount.dev.json
 firebase-debug.log
+.firebase/hosting.YnVpbGQ.cache
```

Comment:
Maybe we could blacklist all the .firebase/*.cache ?

Reply:
Good idea!